src/hooks/useUpdateUserDashboard.ts

```diff
@@ -49,6 +49,7 @@ export default function useUpdateUserDashboard() {
     } else {
       fileRef.set({
         title: settings.workspaceName || '',
+        language: settings.language,
```

Comment:
@thecodingwizard does the version number also need to be changed?

Reply:
**version:2** is already there.

Reply:
ummm honestly I don't even remember when we access .version / why it exists. It's probably fine as is (though it would be nice to figure out what we use / used `.version` for)